tests/src/test/scala/ch/renku/acceptancetests/workflows/Collaboration.scala

```diff
@@ -18,4 +19,35 @@ trait Collaboration {
     Then("they should see a 'No merge requests to display' info")
     verify userCanSee projectPage.Collaboration.MergeRequests.noMergeRequests
   }
+
+  def verifyIssuesIsEmpty(implicit projectDetails: ProjectDetails): Unit = {
+    val projectPage = ProjectPage()
+    When("the user navigates to the Collaboration tab")
+    click on projectPage.Collaboration.tab
+    And("they navigate to the Merge Requests sub tab")
+    click on projectPage.Collaboration.Issues.tab
+    Then("they should see a 'No issues to display' info")
+    verify userCanSee projectPage.Collaboration.Issues.noIssues
```

Comment:
What would you say if we'd do the assertion in the spec/workflow instead of the Page class? What I mean is smth like that:
```projectPage.Collaboration.Issues.noIssues.text shouldBe 'No issues to display.'```
We do have a workflow for the tab so I think it's maybe a better place?

Potentially the `No merge requests to display.` could be done in a similar way...

Reply:
I see what you mean, but it is not clear cut because the structure of the page itself depends on whether or not there are issues.

The test IMHO should check if there is an indication that there are no issues to see, whether the text of that indication matches some expectation is of less importance.

Reply:
👍